### ISSUE TITLE
This demo needs an update

### ISSUE THREAD

**Issue by pdebruin** [Azure-Samples/virtual-kubelet-aci-burst#9]
I tried to follow https://github.com/Azure-Samples/virtual-kubelet-aci-burst, which didnt work. Lots of things have changed:
* The kube-lego chart is gone
* The https://github.com/rbitia/aci-demos repo has changed
* The kubernetes deploymentspec has changed
The scenario still looks relevant. Please update.

**Comment by runemy** [comment]
Yes, would have been nice to have such a demo to show Azure Container Apps.
